app/controllers/rentals_controller.rb

```diff
@@ -41,9 +41,11 @@ def processing
 
   # GET /rentals/1/transform
   def transform
-    if @rental.rental_status == 'reserved'
+    if @rental.reserved? && @rental.end_date < DateTime.current
```

Comment:
Based on what I understood from talking to Jon, the customer will have been charged prior to the rental, thus reserving the cart for the amount of days they wanted. Therefore a customer could conceivably have reserved a cart for multiple days, but not necessarily show up on the first day. Since they'd have already paid for these days, the no-show flag shouldn't occur 'til after the end date.